pkgs/top-level/all-packages.nix

```diff
@@ -9510,7 +9510,7 @@ with pkgs;
 
   ised = callPackage ../tools/misc/ised { };
 
-  isl = isl_0_20;
+  isl = __splicedPackages.isl_0_20;
```

Comment:
This probably wasn't a problem because `isl` would be spliced again

```
nix-repl> pkgsCross.aarch64-multiplatform.__splicedPackages.isl ? __spliced
true
```

Reply:
You're right; it's only the other `isl` that needs it.

I'll push a commit to drop this as soon as ofborg finishes the current eval, which I think might actually succeed!  Don't want to interrupt it just yet.

Reply:
9c46b1a6517b